src/Message/CreatePaymentMethodRequest.php

```diff
@@ -266,6 +271,27 @@ public function setUpdateSubscriptions($value)
         return $this->setParameter('updateSubscriptions', $value);
     }
 
+    /**
+     * If set to false we set the payment method active flag to be false
+     *
+     * @return null|bool
+     */
+    public function getActivatePaymentMethod()
```

Comment:
as discussed offline, parameter function name needs to match the parameter name or Omnipay will fail to initialize it. (see Helper::initialize in Omnipay Common)